cli.ts

```diff
@@ -250,14 +255,18 @@ async function serve(
   }
   const onBuild = () => buildEventHub.dispatchEvent(new CustomEvent("built"));
 
-  const assets = watchAndGenAssets(path, {
-    livereloadPort,
-    onBuild,
-    mainAs404: true,
-  });
+  const allAssets: AsyncGenerator<File, void, void>[] = [];
+  for (const path of paths) {
+    const assets = watchAndGenAssets(path, {
+      livereloadPort,
+      onBuild,
+      mainAs404: true,
```

Comment:
This option means it uses the given path as 404 (not found) page, and if you enable this for multiple entrypoints, they overwrite each other. That's probably confusing to the users. I'd suggest we should enable this only for the first item in `paths`.

Reply:
Makes sense, I'll make that change. Also in some further testing I noticed that if I run this for example:
```sh
```

Reply:
> The index.html from with-simple-assets is overwriting the index.html from simple since it's just using the file name for the output. Any suggestions on how this should be fixed?

I'd suggest we should just throw in that case as first pass. Maybe we can do more sensible handling later if there's any feature request about it.